client/components/swipeable/style.scss

```diff
@@ -0,0 +1,32 @@
+@import '@wordpress/base-styles/mixins';
+.swipeable__container {
+	width: 100%;
+	height: 100%;
+	margin-left: auto;
+	margin-right: auto;
+	position: relative;
+	overflow: hidden;
+	list-style: none;
+	padding: 0;
+	z-index: 1;
+	touch-action: pan-y;
+}
+
+.swipeable__pages {
+	position: relative;
+	width: 100%;
+	height: 100%;
+	z-index: 1;
+	display: flex;
+
+	transition: all 0.5s ease-in-out;
+	transition-property: transform, height;
```

Comment:
Usually transitioning the `height` property is not recommended, as it can cause poor rendering performance due to the fact that every change in `height` can cause layout / reflow

Reply:
This is mostly there to be backwards compatible with how dot pages currently work.

Reply:
Got it. Have you considered removing the transition entirely for the `height` property?

Reply:
I did but then we don't see the height change effect that is currently present on /home/example.com 
Instead, the height doesn't change at all and there is a big whitespace at the bottom of the page.

Reply:
I see. Let's leave this for now, we can always change it later if needed